specs/capella/beacon-chain.md

```diff
@@ -64,6 +76,25 @@ class Validator(Container):
     withdrawn_epoch: Epoch  # [New in Capella]
 ```
 
+#### `BeaconBlockBody`
+
+```python
+class BeaconBlockBody(Container):
+    randao_reveal: BLSSignature
+    eth1_data: Eth1Data  # Eth1 data vote
+    graffiti: Bytes32  # Arbitrary data
+    # Operations
+    proposer_slashings: List[ProposerSlashing, MAX_PROPOSER_SLASHINGS]
+    attester_slashings: List[AttesterSlashing, MAX_ATTESTER_SLASHINGS]
+    attestations: List[Attestation, MAX_ATTESTATIONS]
+    deposits: List[Deposit, MAX_DEPOSITS]
+    voluntary_exits: List[SignedVoluntaryExit, MAX_VOLUNTARY_EXITS]
+    bls_to_execution_changes: List[SignedBLSToExecutionChange, MAX_BLS_TO_EXECUTION_CHANGES]
```

Comment:
if we put `bls_to_execution_changes` at the end of the block, we maintain some binary compatibility with older (by fork) block formats

Reply:
and note, we can still process the creds changes whenever; this comment is just about syntax

Reply:
Yes please additions at the bottom unless replacing an existing property